src/lib/core/ripple/ripple.ts

```diff
@@ -102,11 +107,11 @@ export class MdRipple implements OnChanges, OnDestroy {
 
   /** Ripple configuration from the directive's input values. */
   get rippleConfig(): RippleConfig {
-    return {
+    return extendObject({
       centered: this.centered,
       speedFactor: this.speedFactor,
       radius: this.radius,
       color: this.color
-    };
+    }, this._globalRippleConfig);
```

Comment:
Or to combine disabled & base speed into a global config object (which is completely unrelated to the individual config objects). Though I still question if anyone even wants this feature